core/src/main/java/org/ehcache/core/exceptions/CachePassThroughException.java

```diff
@@ -52,24 +52,24 @@ public CachePassThroughException(final Throwable cause) {
   /**
    * Helper method for handling runtime exceptions.
    * <P>
-   *   Throwing most as {@link CacheAccessException} except for {@code CachePassThroughException}.
+   *   Throwing most as {@link StoreAccessException} except for {@code CachePassThroughException}.
```

Comment:
This class seems to be named weirdly now, given it's relationship with StoreAccessException.